src/Documentation/doc/migrations/ns2.md

```diff
@@ -0,0 +1,57 @@
+# Netscript 2 Migration Guide
+
+With the deprecation of Netscript 1 there are required changes to migrate over.
```

Comment:
Add a hint to whom this migration doc is for, ie.
"... if you have any .script files." If all your scripts are .js you do not need to follow this guide.

Reply:
Good idea, done.